src/Avalonia.Base/Input/TextInput/TextInputReturnKeyType.cs

```diff
@@ -0,0 +1,17 @@
+namespace Avalonia.Input.TextInput;
+
+public enum TextInputReturnKeyType
+{
+    Default,
+    Go,
+    Google,
+    Join,
+    Next,
+    Route,
+    Search,
+    Send,
+    Yahoo,
```

Comment:
Do we need something as specific as Yahoo or Google here? Wouldn't it be enough to use Search instead?

Reply:
Where did we get those values from? Looks looks something OS-specific

Reply:
https://developer.apple.com/documentation/uikit/uireturnkeytype

Reply:
used a small subset that is cross platform.